IngressMaxFields.user.js

```diff
@@ -136,7 +130,7 @@ function wrapper() {
                 self.managePortals(obj, window.portals[x], x);
             }
         }
-        obj.tooMany = obj.count < 50 ? false : true;
+        obj.tooMany = false;
```

Comment:
Yes and will be :)
On May 15, 2016 12:22 PM, "Per Modin" <notifications@github.com> wrote:

> In IngressMaxFields.user.js
> <https://github.com/itayo/IITC-Ingress-Maxfields-Exporter/pull/26#discussion_r63289467>
> :
>
> > @@ -136,7 +130,7 @@ function wrapper() {
> >                  self.managePortals(obj, window.portals[x], x);
> >              }
> >          }
> > -        obj.tooMany = obj.count < 50 ? false : true;
> > +        obj.tooMany = false;
>
> Can't the tooMany references be removed altogether?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/itayo/IITC-Ingress-Maxfields-Exporter/pull/26/files/cea52f104167f7090789ea8e064594e902a8f2f8#r63289467>
>